pages/partners/farmlands.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <main>
+    <div
+      class="bg-cover bg-[#174B33] bg-blend-overlay bg-[url('~/assets/images/countryside.png')]"
+    >
+      <div class="max-w-6xl mx-auto">
+        <div class="text-content-on-color text-base leading-6 uppercase font-semibold tracking-wider pt-16 md:pt-32 pl-4 md:pl-40 pb-3">
```

Comment:
Would be good to touch base with Junjie about the text styling here, could create or reuse a font 👍